detection_rules/devtools.py

```diff
@@ -19,7 +21,7 @@
 from . import rule_loader
 from .eswrap import CollectEvents, add_range_to_dsl
 from .main import root
-from .misc import PYTHON_LICENSE, add_client, client_error
+from .misc import PYTHON_LICENSE, add_client, GithubClient, Manifest, client_error, getdefault
```

Comment:
oh, nvm about the lazily import. this is our already lazy wrapper around GitHub. you're good